components/GridSimplified/GridSimplified.css

```diff
@@ -0,0 +1,41 @@
+.img-res{
+   width:100%;
+   height:100%;
+   
+}
+
+.grid-50-50-layout{
+    width:80%;
+    height:fit-content;
+    display: grid;
+    grid-template-columns: 50% 50%;
+    border:1px solid var(--error--color);
+   
+}
+.grid-70-30-layout{
+    width:80%;
+    height:fit-content;
+    display: grid;
+    grid-template-columns: 70% 30%;
+    border:1px solid var(--error--color);
+   
+}
+
+.grid-30-70-layout{
+    width:80%;
+    height:fit-content;
+    display: grid;
+    grid-template-columns: 30% 70%;
+    border:1px solid var(--error--color);
+   
+}
+.grid-four-layout{
```

Comment:
Hey shrinkhla,

please correct me if I am wrong. We are supposed to make grid-2 and grid-3. You can definitely have additional components like grid-4. But grid-3 is a must, I guess.

Reply:
Yeah,its important as told to us..Thanks for reminding me.